/var/spool/slurmd/job57820/slurm_script: line 11: cd: SSL_VAN: No such file or directory

----------------------------------------------------------------------------
  cuda:
----------------------------------------------------------------------------
     Versions:
        cuda/10.2
        cuda/11.0
        cuda/11.2
        cuda/11.3
        cuda/11.4
        cuda/11.5
        cuda/11.6
        cuda/11.7
        cuda/11.8
        cuda/12.0

----------------------------------------------------------------------------
  For detailed information about a specific "cuda" package (including how to load the modules) use the module's full name.
  Note that names that have a trailing (E) are extensions provided by other modules.
  For example:

     $ module spider cuda/12.0
----------------------------------------------------------------------------

 


CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:136: UserWarning: Found total gpus 2
  warnings.warn("Found total gpus {}".format(args.ngpus_per_node))
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/dictionary.py:196: UserWarning: `threshold_values=True/False` is deprecated, please use `threshold=value` instead.
  warnings.warn("`threshold_values=True/False` is deprecated, please use `threshold=value` instead.")
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/dictionary.py:196: UserWarning: `threshold_values=True/False` is deprecated, please use `threshold=value` instead.
  warnings.warn("`threshold_values=True/False` is deprecated, please use `threshold=value` instead.")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:256: UserWarning: 1 gpu 1
  warnings.warn(f"{args.rank} gpu {args.gpu}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:256: UserWarning: 0 gpu 0
  warnings.warn(f"{args.rank} gpu {args.gpu}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:258: UserWarning: Batch size is: 1 epochs 15000 and task is Task03_Liver
  warnings.warn(f"Batch size is: {args.batch_size} epochs {args.max_epochs} and task is {args.task}")
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/array.py:176: UserWarning: `to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.
  warnings.warn("`to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:324: UserWarning: Total parameters count 56787848
  warnings.warn(f"Total parameters count {pytorch_total_params}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:327: UserWarning: Total args.checkpoint True
  warnings.warn(f"Total args.checkpoint {args.checkpoint}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:335: UserWarning:  Best url model is 64-128-256-512_3-4-6-3_8-8-4-4_vae_inferer_valid_loader_VANV5GL_2_Task03_Liver__best.pt, final model url is 64-128-256-512_3-4-6-3_8-8-4-4_vae_inferer_valid_loader_VANV5GL_2_Task03_Liver__final.pt
  warnings.warn(f" Best url model is {args.best_model_url}, final model url is {args.final_model_url}")
/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/post/array.py:176: UserWarning: `to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.
  warnings.warn("`to_onehot=True/False` is deprecated, please use `to_onehot=num_classes` instead.")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:324: UserWarning: Total parameters count 56787848
  warnings.warn(f"Total parameters count {pytorch_total_params}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:327: UserWarning: Total args.checkpoint True
  warnings.warn(f"Total args.checkpoint {args.checkpoint}")
/home/karimimonsefi.1/SSL_VAN/MSD/main.py:335: UserWarning:  Best url model is 64-128-256-512_3-4-6-3_8-8-4-4_vae_inferer_valid_loader_VANV5GL_2_Task03_Liver__best.pt, final model url is 64-128-256-512_3-4-6-3_8-8-4-4_vae_inferer_valid_loader_VANV5GL_2_Task03_Liver__final.pt
  warnings.warn(f" Best url model is {args.best_model_url}, final model url is {args.final_model_url}")
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:137: UserWarning: Writing Tensorboard logs to ./runs/MSD/test_log
  warnings.warn(f"Writing Tensorboard logs to {args.logdir}")
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:145: UserWarning: 0  Sat Jun  3 11:15:25 2023  Epoch: 0
  warnings.warn(f"{args.rank}  {time.ctime()}  Epoch: {epoch}")
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:145: UserWarning: 1  Sat Jun  3 11:15:25 2023  Epoch: 0
  warnings.warn(f"{args.rank}  {time.ctime()}  Epoch: {epoch}")
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:29: UserWarning: input size torch.Size([6, 1, 96, 96, 96])
  warnings.warn("input size {}".format(data.shape))
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:29: UserWarning: input size torch.Size([6, 1, 96, 96, 96])
  warnings.warn("input size {}".format(data.shape))
[W reducer.cpp:1298] Warning: find_unused_parameters=True was specified in DDP constructor, but did not find any unused parameters in the forward pass. This flag results in an extra traversal of the autograd graph every iteration,  which can adversely affect performance. If your model indeed never has any unused parameters in the forward pass, consider turning this flag off. Note that this warning may be a false positive if your model has flow control causing later iterations to have unused parameters. (function operator())
[W reducer.cpp:1298] Warning: find_unused_parameters=True was specified in DDP constructor, but did not find any unused parameters in the forward pass. This flag results in an extra traversal of the autograd graph every iteration,  which can adversely affect performance. If your model indeed never has any unused parameters in the forward pass, consider turning this flag off. Note that this warning may be a false positive if your model has flow control causing later iterations to have unused parameters. (function operator())
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 0/51  loss: 3.4714  time 25.63s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 1/51  loss: 3.5027  time 0.84s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 2/51  loss: 3.4971  time 0.82s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 3/51  loss: 3.5041  time 0.84s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 4/51  loss: 3.5094  time 0.84s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 5/51  loss: 3.5072  time 0.83s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 6/51  loss: 3.5128  time 0.85s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 7/51  loss: 3.5172  time 0.82s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 8/51  loss: 3.5145  time 0.84s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 9/51  loss: 3.5204  time 0.83s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 10/51  loss: 3.5205  time 0.84s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 11/51  loss: 3.5188  time 5.72s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 12/51  loss: 3.5216  time 0.84s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 13/51  loss: 3.5207  time 0.86s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 14/51  loss: 3.5225  time 0.81s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 15/51  loss: 3.5226  time 0.81s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 16/51  loss: 3.5245  time 0.89s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 17/51  loss: 3.5249  time 0.89s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 18/51  loss: 3.5236  time 0.87s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 19/51  loss: 3.5236  time 8.52s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 20/51  loss: 3.5240  time 0.82s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 21/51  loss: 3.5225  time 0.83s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 22/51  loss: 3.5218  time 0.85s
  warnings.warn(
/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py:51: UserWarning: Epoch 0/15000 23/51  loss: 3.5201  time 0.88s
  warnings.warn(
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/MSD/main.py", line 407, in <module>
    main()
  File "/home/karimimonsefi.1/SSL_VAN/MSD/main.py", line 138, in main
    mp.spawn(main_worker, nprocs=args.ngpus_per_node, args=(args,))
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 240, in spawn
    return start_processes(fn, args, nprocs, join, daemon, start_method='spawn')
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 198, in start_processes
    while not context.join():
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 160, in join
    raise ProcessRaisedException(msg, error_index, failed_process.pid)
torch.multiprocessing.spawn.ProcessRaisedException: 

-- Process 1 terminated with the following error:
Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/multiprocessing/spawn.py", line 69, in _wrap
    fn(i, *args)
  File "/home/karimimonsefi.1/SSL_VAN/MSD/main.py", line 385, in main_worker
    accuracy = run_training(
  File "/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py", line 147, in run_training
    train_loss = train_epoch(
  File "/home/karimimonsefi.1/SSL_VAN/MSD/trainer.py", line 23, in train_epoch
    for idx, batch_data in enumerate(loader):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 628, in __next__
    data = self._next_data()
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1313, in _next_data
    return self._process_data(data)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 1359, in _process_data
    data.reraise()
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/_utils.py", line 543, in reraise
    raise exception
RuntimeError: Caught RuntimeError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 89, in apply_transform
    return _apply_transform(transform, data, unpack_items)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 53, in _apply_transform
    return transform(parameters)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/croppad/dictionary.py", line 1171, in __call__
    self.randomize(label, fg_indices, bg_indices, image)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/croppad/dictionary.py", line 1153, in randomize
    self.centers = generate_pos_neg_label_crop_centers(
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/utils.py", line 512, in generate_pos_neg_label_crop_centers
    centers.append(correct_crop_centers(center, spatial_size, label_spatial_shape, allow_smaller))
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/utils.py", line 439, in correct_crop_centers
    raise ValueError("The size of the proposed random crop ROI is larger than the image size.")
ValueError: The size of the proposed random crop ROI is larger than the image size.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 89, in apply_transform
    return _apply_transform(transform, data, unpack_items)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 53, in _apply_transform
    return transform(parameters)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/compose.py", line 173, in __call__
    input_ = apply_transform(_transform, input_, self.map_items, self.unpack_items, self.log_stats)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 113, in apply_transform
    raise RuntimeError(f"applying transform {transform}") from e
RuntimeError: applying transform <monai.transforms.croppad.dictionary.RandCropByPosNegLabeld object at 0x14ce1a8f3310>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/_utils/worker.py", line 302, in _worker_loop
    data = fetcher.fetch(index)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 58, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/torch/utils/data/_utils/fetch.py", line 58, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/data/dataset.py", line 97, in __getitem__
    return self._transform(index)
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/data/dataset.py", line 83, in _transform
    return apply_transform(self.transform, data_i) if self.transform is not None else data_i
  File "/home/karimimonsefi.1/miniconda3/lib/python3.9/site-packages/monai/transforms/transform.py", line 113, in apply_transform
    raise RuntimeError(f"applying transform {transform}") from e
RuntimeError: applying transform <monai.transforms.compose.Compose object at 0x14ce1a8f1970>


